### PR TITLE
CS-Fixer: Enable `phpdoc_to_property_type`

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -84,12 +84,11 @@
         ],
     ],
     'yoda_style' => false,
+    'php_unit_test_case_static_method_calls' => false,
```
Comment (OskarStark) [inline .php-cs-fixer.dist.php:87]: I would keep this rule, or shall we switch to:

```diff
- $this->assertSame(
+ self::assertSame(
```

**Comment by OskarStark** [comment]
Ready to merge

File: .php-cs-fixer.dist.php (hunk 1 above)
Comment (slunak) [reply .php-cs-fixer.dist.php:87]: Agree, I like `$this->assertSame(` more too.

**Comment by slunak** [comment]
Let me just fix test

**Review by slunak — APPROVED** [review]

**Comment by OskarStark** [comment]
Sorry was too late yesterday and didn't spot them

**Pull request merged**